docs/develop/smart-contracts/custom-improvement-proposals.md

```diff
@@ -17,37 +17,38 @@ Operations are the base components of the Ethereum Virtual Machine (EVM) which
 allow the execution of the smart contract logic. When a developer builds a smart
 contract, the code written in Solidity, or Viper, is not directly interpretable
 by the EVM. Before being able to execute the code in the blockchain, the
-contract has to be compiled via one of the available compilers, like [solc](https://docs.soliditylang.org/en/latest/using-the-compiler.html). The
-compilation converts the human readable contract code into a sequence of operations
+contract has to be compiled via one of the available compilers, like
+[solc](https://docs.soliditylang.org/en/latest/using-the-compiler.html). The
+compilation converts the human-readable contract code into a sequence of operations
 that the virtual machine can interpret and execute
 to perform state transitions or query the latest committed state.
 These operations are called **opcodes**, and are contained
 in a structure called [**jump table**](https://github.com/evmos/evmos/blob/v19.0.0/x/evm/core/vm/jump_table.go#L120-L1094).
 
 Each opcode is defined by specifying the logic that has to be executed when it
 is called inside the EVM, its relationship with the memory, and the gas cost
-associated with it. More specifically, an opcodes is completely defined by:
+associated with it. More specifically, an opcode is completely defined by:
 
-- `SetExecute`: update the execution logic for the opcode.
+- `SetExecute` update the execution logic for the opcode.
 
-- `SetConstantGas`: update the value used for the constant gas cost.
+- `SetConstantGas` update the value used for the constant gas cost.
 
-- `SetDynamicGas`: update the function used to compute the dynamic gas cost.
+- `SetDynamicGas` update the function used to compute the dynamic gas cost.
 
-- `SetMinStack`: update the minimum number of items in the stack required to
+- `SetMinStack` update the minimum number of items in the stack required to
 execute the operation.
 
-- `SetMaxStack`: update the maximum number of items that will be in the stack
+- `SetMaxStack` update the maximum number of items that will be in the stack
 after executing the operation.
 
-- `SetMemorySize`: the memory size required by the operation.
+- `SetMemorySize` the memory size required by the operation.
```

Comment:
how come you changed this? I think it was better before

Reply:
if you're removing the `:` I think the text should be adjusted too, e.g. "`SetMemorySize` the memory size" is bad grammer. The `:` makes this correct

Reply:
Makes sense, I have been led astray from the rabbit...reverted

Reply:
perfect, I think we're done then :)